### PR TITLE
Plot: Maxwell—Jüttner distribution

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/core/convert.py [hunk 1]
```diff
@@ -17,3 +20,30 @@ def convert_to(value: Expr, target_unit: Expr) -> Expr:
...
 def convert_to_float(value: Expr) -> float:
     return float(convert_to(value, S.One))
+
+
+_si_conversions: dict[units.Dimension, Expr] = {
+    dimension_definitions.angle: S.One,
+    dimension_definitions.length: units.meter,
+    dimension_definitions.mass: units.kilogram,
+    dimension_definitions.time: units.second,
+    dimension_definitions.current: units.coulomb,
```
Comment (blackyblack) [inline symplyphysics/core/convert.py:30]: Current is not measured in coulombs.

File: symplyphysics/core/convert.py [hunk 2]
```diff
@@ -17,3 +20,30 @@ def convert_to(value: Expr, target_unit: Expr) -> Expr:
...
+    dimension_definitions.temperature: units.kelvin,
+    dimension_definitions.amount_of_substance: units.mole,
+    dimension_definitions.luminous_intensity: units.candela,
+}
+
+
+def convert_to_si(value: Expr) -> Expr:
+    quantity = value if isinstance(value, Quantity) else Quantity(value)
+    dependencies = dimsys_SI.get_dimensional_dependencies(quantity.dimension)
```
Comment (blackyblack) [inline symplyphysics/core/convert.py:39]: AFAIK this function returns MLT dimensions. So you only need to process mass, length and time.

File: symplyphysics/core/convert.py (hunk 1 above)
Comment (alesanter) [reply symplyphysics/core/convert.py:30]: Yes, it should be _charge_.

Comment (alesanter) [reply symplyphysics/core/convert.py:30]: Of course, confused it with charge

File: symplyphysics/core/convert.py (hunk 2 above)
Comment (alesanter) [reply symplyphysics/core/convert.py:39]: image
Why not, it does support every SI unit.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**